src/css-modules/Navbar.module.css

```diff
@@ -1,13 +1,49 @@
-.oval {
-  width: 2.813rem;
-  height: 2.813rem;
-  margin: 0 0 0 41.5rem;
-  padding: 0.875rem;
-  border: solid 1px #e8e8e8;
-}
-
-.personIcon {
-  width: 1.063rem;
-  height: 1.063rem;
-  background-color: #0290ff;
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;700&display=swap");
+
+* {
+  font-family: 'Montserrat', sans-serif;
+  font-style: normal;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.navbar {
+  background-color: #fff;
+}
+
+.navContainer {
+  margin: 0 6.25rem;
+  display: flex;
+  gap: 2.938rem;
+  align-items: center;
+}
```

Comment:
- Hey, great job with the styling. Please let's kindly improve the padding to add more space to make our design close to the template as much as we can.
- Let's also  add the missing icon to match the design as much as we can below:

*your design*
![Screenshot from 2022-11-19 23-13-11](https://user-images.githubusercontent.com/59397159/202873477-669983d6-7eff-49e7-bf99-8662d19c2db3.png)

*template*
![Screenshot from 2022-11-19 23-12-28](https://user-images.githubusercontent.com/59397159/202873478-071fec0d-8083-4f6d-a5b0-c8fd7c48ee4e.png)

Reply:
Hi @zieeco thanks for giving a review i tried to give icon but its not working and I can't figure out the reason can you help me out in that

Reply:
![required](https://user-images.githubusercontent.com/54817285/202874366-5d8e3cfa-a1d9-4e46-bba8-2cdd0b4db637.png)
done